### PR TITLE
feat(node-shell): optional pod running timeout

### PR DESCRIPTION
**Pull request by maxime1907**
Solves https://github.com/kvaps/kubectl-node-shell/issues/53

Tested with:
```
KUBECTL_NODE_SHELL_POD_RUNNING_TIMEOUT="2m" sh -x kubectl-node_shell kind-control-plane -- ls -all
sh -x kubectl-node_shell kind-control-plane -- ls -all
```

### CODE REVIEW

**Comment by maxime1907** [comment]
cc @kvaps

**Review by klinda01 — APPROVED** [review]

**Comment by maxime1907** [comment]
cc @kvaps could you merge this and release please ?

File: kubectl-node_shell [hunk 1]
```diff
@@ -13,6 +13,7 @@ volumes="[]"
 volume_mounts="[]"
 x_mode=0
 labels="${KUBECTL_NODE_SHELL_LABELS}"
+pod_running_timeout="${KUBECTL_NODE_SHELL_POD_RUNNING_TIMEOUT:=1m}"
```
Comment (kvaps) [inline kubectl-node_shell:16]: Hey @maxime1907,

```suggestion
pod_running_timeout="${KUBECTL_NODE_SHELL_POD_RUNNING_TIMEOUT:-1m}"
```

please update:
- `:=` will set both `pod_running_timeout` and `KUBECTL_NODE_SHELL_POD_RUNNING_TIMEOUT` variables to `1m`, which is somehow unexpected
- `:-` will set `pod_running_timeout` variable to `1m` and leave `KUBECTL_NODE_SHELL_POD_RUNNING_TIMEOUT` empty

Comment (maxime1907) [reply kubectl-node_shell]: Oh i see thanks @kvaps 
I just pushed the required changes!

**Pull request merged**